proposals/csharp-11.0/low-level-struct-improvements.md

```diff
@@ -353,17 +353,21 @@ The language "does not contribute" means the arguments are simply not considered
 
 The method invocation rules can now be simplified. The receiver no longer needs to be special cased, in the case of `struct` it is now simply a `scoped ref T`. The value rules need to change to account for `ref` field returns:
 
-> A value resulting from a method invocation `e1.M(e2, ...)` is *safe-to-escape* from the narrowest of the following scopes:
+> A value resulting from a method invocation `e1.M(e2, ...)`, where M() does not return ref to ref struct, is *safe-to-escape* from the narrowest of the following scopes:
 > 1. The *calling method*
 > 2. When the return is a `ref struct` the *safe-to-escape* contributed by all argument expressions
 > 3. When the return is a `ref struct` the *ref-safe-to-escape* contributed by all `ref` arguments
+> If M() does return ref-to-ref-struct, the *safe-to-escape* is the same as the *safe-to-escape* of all arguments which are ref-to-ref-struct. It is an error if there are multiple arguments with different *safe-to-escape*.
```

Comment:
>ref-to-ref-struct [](http://example.com/codeflow?start=21&length=17)

Minor: We have a mix of "ref to ref struct" and "ref-to-ref-struct" in the added text.